cmd/snap/cmd_set.go

```diff
@@ -155,6 +161,48 @@ func (x *cmdSet) Execute([]string) error {
 	return nil
 }
 
+func validateJsonKeys(v interface{}) error {
+	var initial map[string]interface{}
+	var ok bool
+
+	if initial, ok = v.(map[string]interface{}); !ok {
+		// If it's not a map, there's nothing to validate.
+		// Because, there is no nested map or array.
+		return nil
+	}
```

Comment:
What if it's an array?

Reply:
Very good point! You're absolutely right! So I guess that I will have to address this other edge case:
When the JSON body, it's inside an array.